ubuntu/ml/gpu/rapids/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM sarthfrey/ml-gpu-cuda:latest
+
+RUN /databricks/conda/bin/conda install -n ${DATABRICKS_ROOT_CONDA_ENV} \
+    -c rapidsai -c nvidia -c conda-forge -c defaults \
+    rapids=0.13 python=3.7 cudatoolkit=10.1
```

Comment:
This does not work

Reply:
Have a minimal yml file that describes the rapids.ai dependencies as well as databricks minimal dependencies (ipython, pandas, pyarrow, etc). Then use that yaml to create a new env, which is simpler for conda to resolve dependencies.

Install the latest miniconda (4.8.3), which is faster and better at solving.